jack_mixer.1.rst

```diff
@@ -0,0 +1,68 @@
+=============
+ jack_mixer
+=============
+
+----------------------------------------------
+A graphical multi-channel audio mixer for JACK
+----------------------------------------------
+
+:Author: Nedko Arnaudov <nedko@arnaudov.name> (original author),
+    Frédéric Peters <fpeters@0d.be> (current maintainer).
+:Date: 2020-08-29
+:Copyright: GNU General Pulic License Version 2
```

Comment:
typo: Pulic -> Public

Reply:
Huh, I wonder where I got that from? This is something I would normally copy and paste. But I can't find the same error elsewhere in the repo.

Reply:
Fixed